Server/server.py

```diff
@@ -0,0 +1,60 @@
+import torch
+from flask import Flask, request, jsonify
+from transformers import AutoTokenizer, RobertaTokenizer, RobertaForSequenceClassification, AdamW
+
+# Initialize Flask application
+app = Flask(__name__)
+
+
+def make_predictions(model, texts):
+    # Tokenize the input texts
+    tokenizer = AutoTokenizer.from_pretrained('roberta-base')
+    inputs = tokenizer(texts, truncation=True, padding=True, return_tensors='pt')
+
+    # Move inputs to the same device as the model
+    device = model.device
+    inputs = {key: value.to(device) for key, value in inputs.items()}
+
+    # Make predictions
+    with torch.no_grad():
+        outputs = model(**inputs)
+
+    # Get predicted labels
+    logits = outputs.logits
+    predictions = torch.argmax(logits, dim=1).tolist()
+
+    return predictions
+
+# Load your PyTorch model
+def load_model(model_path):
+    model = RobertaForSequenceClassification.from_pretrained('roberta-base', num_labels=2)
+    model.load_state_dict(torch.load(model_path, map_location=torch.device('cpu')))
+    model.eval()
+    return model
+
+
+# Replace 'model.pth' with the path to your actual model file
+model_path = 'model.pth'
+model = load_model(model_path)
+
+
+# Define the predict endpoint
+@app.route('/predict', methods=['GET'])
```

Comment:
As we use load balancer "Transport layer", I'll modify this to be on the port directly